packages/repository/src/decorators/relation.decorator.ts

```diff
@@ -39,6 +39,12 @@ export interface HasManyDefinition extends RelationDefinitionBase {
   keyTo?: string;
 }
 
+export interface BelongsToDefinition extends RelationDefinitionBase {
+  type: RelationType.belongsTo;
+  keyTo?: string;
```

Comment:
Doesn't either `keyTo` or `keyFrom` need to be required?

Reply:
Yup, as mentioned in https://github.com/strongloop/loopback-next/pull/1618#discussion_r212027927, I think a solution to this is to introduce a intermediate type definition that is used before the relation metadata is resolved